quay.io/pawsey/alphafold/container.yaml

```diff
@@ -0,0 +1,12 @@
+docker: quay.io/sarahbeecroft9/alphafold
+url: https://quay.io/sarahbeecroft9/alphafold
+maintainer: '@sarahbeecroft'
+description: AlphaFold is an artificial intelligence (AI) program developed by Alphabet's/Google's DeepMind which performs predictions of protein structure. This module runs on CPU and NVIDIA GPU only.
+latest:
+  '2.2.3': sha256:cb4af40028e49dd340403b469ff667c924672c5c1ac3caa40eb4f2c680ad74a0
+tags:
+  '2.2.3': sha256:cb4af40028e49dd340403b469ff667c924672c5c1ac3caa40eb4f2c680ad74a0
+aliases:
```

Comment:
This looks good! For the command I would be inclined to leave out bash, and target the script directly (and make sure it has the correct hashbang at the top and is executable). Let me know what you think!